crates/ruff/src/rules/pycodestyle/rules/compound_statements.rs

```diff
@@ -77,7 +79,8 @@ impl Violation for MultipleStatementsOnOneLineSemicolon {
 ///
 /// ## Example
 /// ```python
-/// do_four();  # useless semicolon
+/// do_four()
```

Comment:
Is it intentional to remove the semicolon from this example?

Reply:
No, reverted in e7a89f24a76bd17491f3267e65e01e8054aacc43